app/src/modules/settings/routes/roles/app-permissions.ts

```diff
@@ -278,4 +279,14 @@ export const appMinimalPermissions: Partial<Permission>[] = [
 			'role',
 		],
 	},
+	{
+		collection: 'directus_users',
+		action: 'update',
+		permissions: {
+			id: {
+				_eq: '$CURRENT_USER',
+			},
+		},
+		fields: ['tfa_secret'],
+	},
```

Comment:
> Ok, then I'll close this one. Is there an easy way to see pending PR's on a specific file?

I wish that too, but, unfortunately, didn't find anything yet.

> Will PR #7805 be merged shortly you think? Otherwise it might still be usefull to merge this one. It's a small change, but it took me hours to find out why one of my users couldn't enable 2FA. So would be nice if we could prevent this from happening again to someone else.

[**Unfortunately, we can not provide estimates on features or fixes**](https://docs.directus.io/getting-started/support.html#can-you-give-an-eta-for-this-feature-fix)

That being said, we can leave this open, but, meanwhile, I think this chunk of code should be removed.
IMO, it does not make sense to not be able to update the password, but be able to update 2FA secret.

Reply:
> I think this chunk of code should be removed.

Well, even for users with minimal permissions the option to enable 2FA is available. But after the user scanned the QR and entered the OTP for the firsttime a permission denied exception is thrown. 

> it does not make sense to not be able to update the password, but be able to update 2FA secret.

I fully agree! But the changes in the PR were solely meant to fix #9883. 

> Also, is it possible to make your PR a bit cleaner? You can do a rebase and pick only the commits you want, or, just reset hard from main and commit only the necessary and force push into this branch.

Thanks the feedback! Still learning to find my way in Github...